src/components/Wizard/FormStep.tsx

```diff
@@ -0,0 +1,63 @@
+import { Block } from "baseui/block";
+import { Button } from "baseui/button";
+
+export function FormStep<ComponentProps = {}> ({
+  title,
+  description,
+  status = "future",
+  Component,
+}: StepProps<ComponentProps>) {
+  return null;
+};
+
+export function StepLayout<ComponentProps = {}> ({
+  previous,
+  next,
+  Component,
+  values,
+  setValues,
+  ...componentProps
+}: StepLayoutProps<ComponentProps>) {
+  const onSubmit = (arg: any) => {
+    setValues({ ...values, ...arg });
+    next();
+  };
+  return (
+    <Block
+      display="flex"
+      alignItems="flex-start"
+      justifyContent="flex-start"
+      flexDirection="column"
+    >
+      {!!previous ? (
```

Comment:
Instead of doing a ternary in the return expression, what do you think about having a variable above that holds null by default, and if previous is truthy, you set it to the Button? Then this simplifies to `{ previous }` in the return statement. Slightly easier to parse than a ternary.

Reply:
Or, if you really want to keep it all in the return, you could just &&

```
{ !!wallet_connected && (<Header showWalletConnect={true} /> )}
```

Reply:
This has been updated to return an empty div for layout purposes in case of no previous

Reply:
My primary comment was in reference to having if/else branching within the return statement. For me at least it's kinda messy and challenging to parse rather then using branching above the return with an if statement thereby keeping the return statement more explicit with less branching. You're fine to keep it how you want in this case. I know it's not uncommon with React, I just wanted to clarify my reasoning.